### PR TITLE
exit watchdog on windows on clean stop

### PR DESCRIPTION
**Pull request by zanderz**
This has bugged me for a really long time and users have complained about it. Related to DESKTOP-5956, which will be blocked on merging this. BTW, tried this in a couple of production builds, including updating, etc.
@keybase/updater-hackers @maxtaco

### CODE REVIEW

File: watchdog/watchdog.go [hunk 1]
```diff
@@ -79,6 +82,10 @@ func watchProgram(program Program, restartDelay time.Duration, log Log) {
 			if program.ExitOn == ExitOnSuccess {
 				log.Infof("Program configured to exit on success, not restarting")
 				break
+			} else if program.ExitOn == ExitAllOnSuccess {
+				log.Infof("Program configured to exit on success, exiting")
+				// presumably the other wathches are sleeping anyway
```
Comment (jzila) [inline watchdog/watchdog.go]: s/wathches/watchers

Comment (zanderz) [reply watchdog/watchdog.go]: heh, whoops

**Comment by zanderz** [comment]
how about now?

**Comment by zanderz** [comment]
@keybase/updater-hackers @maxtaco @keybase/react-hackers

**Pull request merged**